src/scripts/generate_covid_case_datamart.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import pathlib
+import json
+from typing import List, Callable
+from utils import read_blob, record_combination_func, write_blob
+
+
+def standardize_lab_result(result: str) -> str:
+    """
+    Given a covid test result and dictionary whose keys (POSITIVE, NEGATIVE, and
+    INCONCLUSIVE) represent standard results and whose values are lists containing
+    non-standard results that map to the associated standard result then return a
+    standardized result. Return null is no standard result could not be found.
```

Comment:
smallest possible nit: you mean `if no` but have `is no`, I'll see myself out 😂

Reply:
fixed.